lib/event_sourcery/event_processing/error_handlers/constant_retry.rb

```diff
@@ -1,14 +1,21 @@
 module EventSourcery
   module EventProcessing
     module ErrorHandlers
+      # Module which includes multiple ways of error handling.
```

Comment:
Maybe something like "Strategies for error handling" or "Error handling strategies"?